pkl-cli/pkl-cli.gradle.kts

```diff
@@ -156,7 +156,7 @@ fun Exec.configureExecutable(isEnabled: Boolean, outputFile: File, extraArgs: Li
         ,"--no-fallback"
         ,"-H:IncludeResources=org/pkl/core/stdlib/.*\\.pkl"
         ,"-H:IncludeResources=org/jline/utils/.*"
-        ,"-H:IncludeResources=org/pkl/commons/cli/commands/IncludedCARoots.pem"
+        ,"-H:IncludeResources=org/pkl/core/http/IncludedCARoots.pem"
```

Comment:
The resource is now exclusively used by `pkl.core.http.HttpClientBuilder`, so moving it was natural. I believe it is also necessary because core doesn't depend on commons-cli.